gaeun/app.js

```diff
@@ -44,8 +44,7 @@ app.get("/ping", (req, res) => {
 // Assignment2 - 유저 회원가입 //
 //////////////////////////////
 
-// http -v POST 127.0.0.1:3000/user/signup name="Troye Sivan" email="troye@gmail.com" password="troyepw" profileImage="ts_profile_image.url"
-app.post("/user/signup", async (req, res) => {
+app.post("/users", async (req, res) => {
```

Comment:
```suggestion
app.post("/users/signup", async (req, res) => {
```
정확하게 주소를 작성해주세요~ `/users`만으로는 해당 엔드포인트가 정확히 무슨 기능을 하는지 알 수 없을 뿐더러 users를 사용하여 수많은 유저 관련 엔드포인트가 생성될 수 있음을 생각한다면 더욱 디테일하게 작성해야 합니다!